README.md

```diff
@@ -23,6 +23,8 @@ Refer to [INSTALL.md](INSTALL.md)
 
 Make sure `bitcoind` is running before running `teosd` (it will fail at startup if it cannot connect to `bitcoind`). [Here](DEPENDENCIES.md#installing-bitcoind) you can find a sample bitcoin.conf.
 
+Please see [Docker instructions](docker/README.md) for instructions on how to set up teos in Docker.
```

Comment:
:s/teos/`teosd`

Reply:
oh what does :s mean? I assume this means change "teos" to teosd"?

Reply:
yup it means `substitute`